rhobs/alerting/data_plane/prometheus.gitops_cluster_agent_op_alerts.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-cluster-agent-operations-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: cluster-agent-operations_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsClusterAgentOperationErrors
+      expr: |
+        sum(operationDB_rows_in_non_complete_state) <= 10
```

Comment:
Do we need to use `sum` here? Looking at AppSRE Grafana, I see a single data series per cluster for this metric.
Using sum here means that we're summing over multiple clusters, which I'm not sure is the intention.

Reply:
I have added a comment above regarding this https://github.com/redhat-appstudio/o11y/pull/117#discussion_r1399010413 please let me know your suggestions as well